### PR TITLE
Fix for issues/43 - postgresql Insert() not works

### PR DESCRIPTION
**Pull request by LiveLikeLastDay**
https://github.com/henkmollema/Dommel/issues/43

### CODE REVIEW

File: src/Dommel/DommelMapper.cs [hunk 1]
```diff
@@ -1486,7 +1486,7 @@ private sealed class PostgresSqlBuilder : ISqlBuilder
         {
             public string BuildInsert(string tableName, string[] columnNames, string[] paramNames, PropertyInfo keyProperty)
             {
-                var sql = string.Format("insert into {0} ({1}) values ({2}) select last_insert_rowid() id",
+                var sql = string.Format("insert into {0} ({1}) values ({2})",
```
Comment (henkmollema) [inline src/Dommel/DommelMapper.cs]: You just completely removed the select statement for the ID. Didn't you mean to add a `select LASTVAL()` or use a `RETURNING` statement?

**Comment by henkmollema** [comment]
@LiveLikeLastDay see my comments.

**Comment by Kemyke** [comment]
I think this PR is fine and should be merged. 
It is enough to use RETURNING to get the inserted id and you don't need the "select LASTVAL()" statement. With this modification the insertation is successful.

**Comment by henkmollema** [comment]
Ah now I see. The `returning` statement is added after this line. Silly me. :confused: 

I'll see if I can merge this, there are some conflicts though.

**Comment by henkmollema** [comment]
@LiveLikeLastDay @Kemyke fixed with 5febf88.

**Comment by Kemyke** [comment]
Ok, if you have some merge problems i can make a PR with my implementation of the PostgresSqlBuilder which is a little bit refactored and simplified version of yours.

**Comment by henkmollema** [comment]
I managed to merge it. Pushed v1.7.1: https://www.nuget.org/packages/Dommel/1.7.1

**Pull request closed without merging**